src/modules/fancyzones/lib/util.cpp

```diff
@@ -25,3 +25,85 @@ UINT GetDpiForMonitor(HMONITOR monitor) noexcept
 
     return (dpi == 0) ? DPIAware::DEFAULT_DPI : dpi;
 }
+
+void OrderMonitors(std::vector<std::pair<HMONITOR, RECT>>& monitorInfo)
+{
+    // blocking[i][j] - whether monitor i blocks monitor j in the ordering, i.e. monitor i should go before monitor j
+    std::vector<std::vector<bool>> blocking(monitorInfo.size(), std::vector<bool>(monitorInfo.size(), false));
```

Comment:
nit: let's just extract `monitorInfo.size()` in the begining of the function as local variable, as it is used bunch of times later on

Reply:
I was thinking about that, but any descriptive name would be almost as long as `monitorInfo.size()`. Perhaps `nMonitors`?

Reply:
I guess `nMonitors` sounds about right.